packages/opentelemetry/lib/tracing/exporter.js

```diff
@@ -0,0 +1,31 @@
+const jagger = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
+const { ConsoleSpanExporter } = require('@opentelemetry/tracing');
+
+const DEFAULT_SERVICE_NAME = 'appium';
+
+const available_exporters_with_default_config = {
+  JAGGER: 'jagger',
+  ZIPKIN: 'zipkin',
+  PROMETHEUS: 'prometheus',
+  CONSOLE: 'console'
+};
+
+const build_exporter = function (exporter_type, config = undefined) {
```

Comment:
Please prefer camelCase

`config = undefined` does not make much sense to me

Reply:
does it make sense to use anonymous function here?

Reply:
yeah not a problem, general practice, usually easier for other people from different languages to read code because its more explicit. But can change

Reply:
it might be hard to debug stacktraces of such functions. I would prefer each "classic" definition: `function foo (args)` or `const funcName = function funcName (args)` if it should be a variable.

Reply:
anonymous functions are ok if they are small/local nested code blocks

Reply:
also, it is important o mention the difference between undefined and null. undefined means the variable is not defined and null means it is set to no value. I prefer to use `arg = null` notation to reflect the fact that the particular argument is not required